tests/cli_tests/test_cli_start.py

```diff
@@ -10,6 +10,13 @@ def test_start_on_connection_error():
     cli.close()
 
 
+def test_start_with_client_name():
+    cli = pexpect.spawn("iredis --client_name iredis", timeout=1)
+    cli.sendline("CLIENT GETNAME")
+    cli.expect(r"iredis")
```

Comment:
I think this testcase will always pass. 
Since there is "iredis" char when iredis starting.

![image](https://user-images.githubusercontent.com/9675939/118598256-5c0f5c80-b7e0-11eb-88e8-76ec938a1d64.png)

Reply:
Yes, It's a wrong case.  I will fix it